app/uk/gov/hmrc/gform/models/FastForward.scala

```diff
@@ -62,7 +63,17 @@ object FastForward {
 
   case object Yes extends FastForward
   case class StopAt(stopAt: SectionNumber) extends FastForward
-  case class CYA(from: SectionNumber, to: Option[SectionNumber] = None) extends FastForward // {
+  case class CYA(from: SectionNumber, to: Either[CYA.Summary, SectionNumber] = Left(CYA.FinalSummary))
```

Comment:
> Can we provide a better representation for this rather than `Either`?

something like? :
```
sealed trait SectionOrSummary extends Product with Serializable
case class Section(sectionNumber: SectionNumber) extends SectionOrSummary
case object FormSummary extends SectionOrSummary
case object TaskSummary extends SectionOrSummary
```

Reply:
> Also why
> 
> ```
> case object Yes extends FastForward
> ```
> 
> still exists, what is its purpose?
> 
> It feels weird when you enter the form, and fastForward is thus disable to have `ff=t` in the url thinking

`FastForward.Yes` still represent the logic on how to progress to the next section. It kind of represents a "normal" workflow without `Back` and jumping from `CYA` pages. 
I probably should've rename it.

Reply:
When I enter form (assume most basic form with 4 pages) I can see `ff=t` in the url, when I click back link on second page it takes me to first page and url changes to `ff=1` so far so good. But as I continue in the journey value of `ff` param is always a number.

That doesn't seem to correspond with

> `FastForward.Yes` still represent the logic on how to progress to the next section. 

So I wonder can we simple get rid of it?

Reply:
> So I wonder can we simple get rid of it?
Can we do that in a separate ticket?  I think Removing`Yes not straightforward.

Reply:
Ok

Reply:
> Yes

https://jira.tools.tax.service.gov.uk/browse/GFORMS-1981